concordium-cis2/src/lib.rs

```diff
@@ -854,51 +861,6 @@ impl<T: IsTokenId, A: IsTokenAmount> schema::SchemaType for Cis2Event<T, A> {
     }
 }
 
-impl<T: IsTokenId, A: IsTokenAmount> Serial for Cis2Event<T, A> {
-    fn serial<W: Write>(&self, out: &mut W) -> Result<(), W::Err> {
-        match self {
-            Cis2Event::Transfer(event) => {
-                out.write_u8(TRANSFER_EVENT_TAG)?;
```

Comment:
Could you not clean up the constants as well?

Reply:
You mean remove them? Unfortunately, we still have some manual implementations left where the constants are useful.

Reply:
Ok then. :)